### ISSUE TITLE
postgres + extensions

### ISSUE THREAD

**Issue by pjebs** [kubedb/project#423]
Is there a way to install custom postgres extensions such as timescaledb?

**Comment by hossainemruz** [comment]
@pjebs Yes. Check this guide: https://kubedb.com/docs/0.9.0/guides/postgres/custom-versions/setup/

**Comment by gboor** [comment]
Would it be an idea to include the most common extensions in the base kubedb postgres image? Like PostGIS and Timescale, for starters, while also providing a framework/template for adding more plugins. This allows keeping in line with the kubedb release images.

I see no downside to this; you don't HAVE to enable the plugins. Using the custom image approach, all we end up with is hundreds of images doing the same thing spread across various public and private registries.

**Comment by vaxinate** [comment]
I am interested in making some contributions back to the kubedb/postgres-docker repo to enhance support for postgis and timescale as well as newer versions of postgres. @hossainemruz if I were to fork that repo and begin making enhancements to your dockerfiles would your team be willing to accept pull requests?
